PhysicsTools/PatAlgos/plugins/PATObjectCrossLinker.cc

```diff
@@ -0,0 +1,229 @@
+// -*- C++ -*-
+//
+// Package:    PhysicsTools/NanoAOD
```

Comment:
this doesn't match the file location.
full path given in the line with ```\class ```  doesn't match either.